config/routes.rb

```diff
@@ -230,7 +230,6 @@
     resource :inbox, controller: "inbox" do
       member do
         get :reply
-        get :cancel_reply
```

Comment:
This route was introduced in d142b55ad7640fbd6d03805a7204f3123904483d. It has never had an action.